### PR TITLE
[Feature] Seperate Global and Channel BetterTTV emotes

### PR DESCRIPTION
**Pull request by dclstn**

### CODE REVIEW

File: src/modules/emote_menu/stores/index.js [hunk 1]
```diff
@@ -73,14 +73,23 @@ class EmoteStore extends SafeEventEmitter {
...
+      twitch.getCurrentChannelProfilePicture(),
+    ]);
 
     emoteProviderCategories = [
       createCategory(
-        emotesCategoryIds.BETTERTTV,
-        'BetterTTV',
+        emotesCategoryIds.BETTERTTV_CHANNEL,
+        'BetterTTV Channel',
+        Icons.IMAGE(channelPicture == null ? cdn.url('/assets/logos/mascot.png') : channelPicture, 'BetterTTV'),
```
Comment (night) [inline src/modules/emote_menu/stores/index.js:85]: it is confusing to use the same image for the channel all over the place. we need to have distinguishable icons

Comment (dclstn) [reply src/modules/emote_menu/stores/index.js:85]: any ideas on what the image could be instead?

Comment (night) [reply src/modules/emote_menu/stores/index.js:85]: maybe there's some way we can build a component which renders the channel icon with a provider icon overlaid in the bottom right corner.

**Comment by night** [comment]
superseded by cbdb9e03a24df6d04078503b0c133ce8c95c8d1d

**Pull request closed without merging**